### PR TITLE
feat(console): set email verified on org creation, disable svg upload, password page optimizations

### LINKED ISSUE #3913: Changing Password Requirements - New User
- [x] Get rid of 'The Password' in the hints (redudancy)
- [x] place the hints below the passwort input fields
- [x] Set Primary Button to the left
- [x] set 2 blocks of hints closer to each other for easier readability

See arrangement below (for exact setup)

Image

### LINKED ISSUE #3923: General Button Placement left aligned

### LINKED ISSUE #3952: [console] add verified email flag on create org
If i want to create a new organisation including a new user i'm not able to set the email of the user verified.

**To Reproduce**
1. Create new org
2. set some name and domain
3. try to set the email of the user verified

**Expected behavior**

I should be able to set the email verified.

**Screenshots**

how it should look:
image

how it looks:
image

### LINKED ISSUE #4092: Bug: Branding Logo can upload SVG, but will not be displayed
Despite the text, I can upload a SVG as a logo in the 'branding options' and even get a success message, but it won't display.

Image

We should not allow a SVG upload in the first place and only allow the desired file formats with a proper error message back to the user.

### PR DESCRIPTION
**Pull request by peintnermax**
Closes #3952
Closes #4092
Closes #3913
Closes #3923

### CODE REVIEW

**Comment by livio-a** [comment]
we should disable svg upload in the backend as well

**Comment by hifabienne** [comment]
@peintnermax is this ready for review? please do assign someone, so this person knows that he should review

File: console/src/assets/i18n/it.json [hunk 1]
```diff
@@ -540,12 +540,12 @@
...
+      "MINLENGTH": "Deve essere lunga almeno {{requiredLength}} caratteri.",
       "NOEMAIL": "Il nome utente non pu\u00f2 essere un indirizzo e-mail.",
-      "UPPERCASEMISSING": "La password deve includere un carattere maiuscolo.",
-      "LOWERCASEMISSING": "La password deve includere un carattere minuscolo.",
-      "SYMBOLERROR": "La password deve includere un simbolo o un segno di punteggiatura.",
-      "NUMBERERROR": "La password deve includere una cifra."
+      "UPPERCASEMISSING": "Deve includere un carattere maiuscolo.",
+      "LOWERCASEMISSING": "Deve includere un carattere minuscolo.",
+      "SYMBOLERROR": "Deve includere un simbolo o un segno di punteggiatura.",
+      "NUMBERERROR": "Deve includere una cifra."
```
Comment (livio-a) [inline console/src/assets/i18n/it.json:548]: there are some more translations with `La password deve...`

File: console/src/assets/i18n/fr.json [hunk 2]
```diff
@@ -1003,6 +1003,7 @@
       "MAXSIZE": "La taille maximale est limitée à 524 Ko.",
       "EMAILNOSVG": "Le format de fichier SVG n'est pas supporté dans les emails. Téléchargez donc votre logo en PNG ou dans un autre format pris en charge.",
       "MAXSIZEEXCEEDED": "La taille maximale de 524kB est dépassée.",
+      "NOSVGSUPPORTED": "SVG n'est pas pris en charge",
```
Comment (livio-a) [inline console/src/assets/i18n/fr.json:1006]: there are some `Le mot de passe doit...` which need to be changed

File: console/src/assets/environment.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
-  "api": "https://bringmos-jv1jqo.zitadel.app",
-  "issuer": "https://bringmos-jv1jqo.zitadel.app",
-  "clientid": "163706959596355841@zitadel",
-  "customer_portal": "https://zitadel.cloud"
+  "api": "https://bringmos-dev-mfhquc.zitadel.cloud",
+  "issuer": "https://bringmos-dev-mfhquc.zitadel.cloud",
+  "clientid": "168811977588670721@zitadel",
+  "customer_portal": "https://zitadel.cloud/admin/instances"
```
Comment (livio-a) [inline console/src/assets/environment.json:5]: we should change this to localhost and then add it to .gitignore afterwards so we don't have a change everytime

**Review by livio-a — CHANGES_REQUESTED** [review]
as state before, we should disable svg in the api as well or might be even better to explicitly allow certain types, e.g. png, jp(e)g, tif(f)
@juergrinaldi any opinion on this?

File: console/src/assets/i18n/it.json (hunk 1 above)
Comment (peintnermax) [reply console/src/assets/i18n/it.json:548]: changes missing ones

File: console/src/assets/i18n/fr.json (hunk 2 above)
Comment (peintnermax) [reply console/src/assets/i18n/fr.json:1006]: done

File: console/src/assets/environment.json (hunk 3 above)
Comment (peintnermax) [reply console/src/assets/environment.json]: changed it to localhost, but the .gitignore has no effect as long as the file is checked in.. 🤔

**Comment by juergrinaldi** [comment]
> as state before, we should disable svg in the api as well or might be even better to explicitly allow certain types, e.g. png, jp(e)g, tif(f) @juergrinaldi any opinion on this?

Well, since SVG are lossless and not resolution dependend, I'd advocate for allowing them if it's not too much of a hassle. other than that, I guess PNG/JPG are enough.

**Comment by fforootd** [comment]
@peintnermax whats the state of this PR, since it closes 4 issues I think we should move on with this

**Comment by fforootd** [comment]
> We decided to provide an attribute from the backend where allowed formats are defined. Just like supportedLanguages

Ok and who is implementing this?

**Comment by peintnermax** [comment]
discussed it with @livio-a. required task is the following https://github.com/zitadel/zitadel/issues/4256

**Comment by fforootd** [comment]
@livio-a do we close that issue with this PR?

**Comment by livio-a** [comment]
ok let's just merge this one and handle the assets later on (i've updated #4256 accordingly)

@peintnermax can you quickly solve the conflict / remerge main, please

File: console/src/assets/environment.json (hunk 3 above)
Comment (peintnermax) [reply console/src/assets/environment.json]: the file has been removed entirely now

**Review by livio-a — APPROVED** [review]

**Pull request merged**